### PR TITLE
chore(repo): Improve contributing docs

### PR DESCRIPTION
**Pull request by dimkl**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [x] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

### TL;DR

- Move SECURITY.md in root level /docs and use it for all packages
- Move CONTRIBUTING.md in root level /docs and use it for all packages
- Move CODE_OF_CONDUCT.md in root level /docs and use it for all packages
- Improve and use docs/PUBLISH.md as reference for publishing across all packages

**REVIEW IT PER COMMIT**

### CODE REVIEW

**Review by nikosdouvlis — COMMENTED** [review]
:broom

**Review by nikosdouvlis — APPROVED** [review]
🧹
